StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/ClickableLabelNew.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  ClickableLabelNew.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 5/2/23.
+//
+
+import Foundation
+import SafariServices
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+final class ClickableLabelNew: HitTestView {
```

Comment:
This should be nearly equivalent of `ClickableLabel` (pure copy paste) except for the change from `UIFont` to `FinancialConnectionsFont`

Reply:
The goal of the copy-paste is to create the ability to do a piece-by-piece refactor.

All of these changes will be merged into a feature branch, and when done, we should be able to delete `ClickableLabel`